docs/content/docs/test/Runner.md

```diff
@@ -17,48 +17,205 @@ It uses an internal TMS (test management system). We are working on support othe
 
 ## Getting started
 
-TBD
-
-## Configuration
-
-A configuration is done in gradle script:
+1. {{< expand "Apply the gradle plugin" >}}
+Add to your build.gradle.kts
 
 ```kotlin
 plugins {
     id("com.android.application")
     id("com.avito.android.instrumentation-tests")
 }
+```
+
+{{< /expand >}}
+
+2. {{< expand "Add common plugin configuration" >}}
 
+```kotlin
 extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
-    // all options goes here
+    // they are required for Avito app. We will make them optional in future.
+    reportApiUrl = "http://stub"
+    reportApiFallbackUrl = "http://stub"
+    reportViewerUrl = "http://stub"
+    registry = "registry"
+    sentryDsn = "http://stub-project@stub-host/0"
+    slackToken = "stub"
+    fileStorageUrl = "http://stub"
+
+    configurationsContainer.register("Local") {
+        tryToReRunOnTargetBranch = false
+
+        targetsContainer.register("api27") {
+            deviceName = "API27"
+
+            scheduling = SchedulingConfiguration().apply {
+                quota = QuotaConfiguration().apply {
+                    retryCount = 1
+                    minimumSuccessCount = 1
+                }
+
+                reservation = TestsBasedDevicesReservationConfiguration().apply {
+                    device = LocalEmulator.device(27)
+                    maximum = 1
+                    minimum = 1
+                    testsPerEmulator = 1
+                }
+            }
+        }
+    }
 }
-
 ```
 
-### Test APK
+{{< /expand >}}
 
-Plugin builds APKs on his own by default.\
-If for any reason you have to build APK externally, you can pass files manually:
+3. {{< expand "Run tests via gradle task" >}}
 
-```kotlin
-// optional
-applicationApk = "/path/to/app.apk"
-// optional
-testApplicationApk = "/path/to/test-app-debug-androidTest.apk"
+```shell script
+    ./gradlew :<projectPath>:instrumentation<ConfigurationName>           
 ```
 
-### Tests filtering
+In our case:
+
+```shell script
+    ./gradlew :<projectPath>:instrumentationLocal 
+```
 
-TBD
+{{< /expand >}}
 
 ## Examples
 
 Check out a configuration to run in `GradleInstrumentationPluginConfiguration` in the [test app](https://github.com/avito-tech/avito-android/blob/develop/subprojects/android-test/test-app/build.gradle.kts#L114)
 
-### How to run tests on local emulator
+## How to
+
+Firstly you must apply a plugin TODO Getting started ref
+
+### Run tests on local emulator
 
 1. Run an emulator with 27 API
-1. Run `./gradlew :subprojects:android-test:test-app:instrumentationLocal`
+2. Run `./gradlew :subprojects:android-test:test-app:instrumentationLocal`
+
+### Filter tests for execution
```

Comment:
```suggestion
### Filtering tests for execution
```

Reply:
Could you explain why I need to prefer `Filtering`.
For me "How to Filter tests for execution" looks better than "How to Filtering tests for execution"

Reply:
"How to Filtering tests for execution"

Of course, In one sentence it's incorrect. But I don't read paragraphs combining all previous to one cumulative text. I just read what I see on the screen.

Example:
<img width="1009" alt="Screenshot 2020-04-20 22 07 33" src="https://user-images.githubusercontent.com/1104540/79789907-9a271c00-8353-11ea-8f6e-257a8c9e17ec.png">

How I read these paragraphs:
- Write the code - ok, take a look at writing the code. What the first?
- Add the main() method using live templates - I see, it will be my next task

I think we should avoid the vague term "How to". As for me, the whole documentation is one giant how to.

What I propose: rename "How to" to "Configuration\Customization\..."

Reply:
Let discuss it by voice. Because I have different idea